libadaptivity/load_balance/include/comTools.h

```diff
@@ -215,18 +217,19 @@ void allSendRecv(const std::vector< std::vector<T> >& sendBuffer,
   for(size_t p=0;p<NProcs;p++){
     send_count[p] = sendBuffer[p].size();
   }
-  MPI::COMM_WORLD.Alltoall(&(send_count[0]), 1, MPI::INT,
-                           &(recv_count[0]), 1, MPI::INT);
+  MPI_Alltoall(&(send_count[0]), 1, MPI_INT,
+                           &(recv_count[0]), 1, MPI_INT, MPI_COMM_WORLD);
   
   // MPI_Status sendStatus, recvStatus;
-  std::vector<MPI::Request> sendRequest(NProcs, MPI::REQUEST_NULL);
-  std::vector<MPI::Request> recvRequest(NProcs, MPI::REQUEST_NULL);
-  MPI::Status recvStatus;
+  std::vector<MPI_Request> sendRequest(NProcs, MPI_REQUEST_NULL);
+  std::vector<MPI_Request> recvRequest(NProcs, MPI_REQUEST_NULL);
+  MPI_Status recvStatus;
   
   // Send
   for(size_t p=0;p<NProcs;p++){
     if(send_count[p]){
-      sendRequest[p]=MPI::COMM_WORLD.Isend(&(sendBuffer[p][0]), send_count[p], const_mpi_type, p, 1);
+      //sendRequest[p]=MPI_COMM_WORLD.Isend(&(sendBuffer[p][0]), send_count[p], const_mpi_type, p, 1);
+      MPI_Isend(&(sendBuffer[p][0]), send_count[p], const_mpi_type, p, 1,MPI_COMM_WORLD, &sendRequest[p]);
```

Comment:
I see they are't needed, but brackets around the new reference might make it less ambiguous.